### ISSUE TITLE
Coastal Vulnerability sample data missing SLR vector

### ISSUE THREAD

**Issue by emlys** [natcap/invest#452]
There isn't a sample SLR vector in the sample data. It's optional but would be nice to have for testing.

**Comment by dcdenu4** [comment]
I agree that we should ship sample data for all optional inputs as well. This seems like a nice quality of life thing from a users perspective who is getting familiar with the model.

**Comment by davemfish** [comment]
I had some back-and-forth on this with the Marine team when the CV sample data was being assembled and this is where we ended up. I agree it would be better to have something than nothing. If we want it to be meaningful (not totally made-up values) then we ought to get something from Jess or maybe Stacie. If we want to just make some pretend data, I think that's okay too, but the Marine team should be informed so they are not surprised to find bogus data next time they're doing a training.
